src/modules/Library/features/Popups/AddToBuilderPopup/AddToBuilderPopup.tsx

```diff
@@ -219,7 +220,7 @@ export const AddToBuilderPopup = ({
     >
       <>
         {isSpinnerVisible && <Spinner uiType={SpinnerUiType.Secondary} noBackground />}
-        <StyledModalWrapper>
+        <StyledModalWrapper data-testid={`library-cart-add-to-builder-popup-step-${step}`}>
```

Comment:
Could you please move "library-cart-add-to-builder-popup" to the variable and use it in all data-testid properties in this component?